### PR TITLE
Remove extra "preview" word

### PR DESCRIPTION
**Pull request by Youssef1313**
The preview version of VS for Mac now is 8.7, and the current release is 8.6.
So, 8.4 is no longer preview.

### CODE REVIEW

File: docs/core/porting/winforms.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ In this article, various names are used to identify types of files used for migr
 - A working Windows Forms project in a solution that builds and runs without issue.
 - A project coded in C#.
 
-> [!NOTE]
+> [!NOTE] <!-- TODO: (Need guidance on how this note should be changed) -->
```
Comment (Youssef1313) [inline docs/core/porting/winforms.md:33]: @OliaG, Any recommendation is this note?
It was added from a blog of you when version 16.5 was still in preview, that's outdated now.
Should the note be completely removed? or you do have another recommendation?

File: docs/core/porting/winforms.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ In this article, various names are used to identify types of files used for migr
 
 ## Prerequisites
 
-- [Visual Studio 2019 16.5 Preview 1](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) or later for any designer work you want to do. We recommend to update to the latest [Preview version of Visual Studio](https://visualstudio.microsoft.com/vs/preview/).
+- [Visual Studio 2019 16.5 or later](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) or later for any designer work you want to do. We recommend to update to the [latest version of Visual Studio](https://visualstudio.microsoft.com/vs/).
```
Comment (gewarren) [inline docs/core/porting/winforms.md:23]:
```suggestion
- [Visual Studio 2019 version 16.5 or later](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) or later for any designer work you want to do. We recommend to update to the [latest version of Visual Studio](https://visualstudio.microsoft.com/vs/).
```

File: docs/core/porting/winforms.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ In this article, various names are used to identify types of files used for migr
 
 ## Prerequisites
 
-- [Visual Studio 2019 16.5 Preview 1](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) or later for any designer work you want to do. We recommend to update to the latest [Preview version of Visual Studio](https://visualstudio.microsoft.com/vs/preview/).
+- [Visual Studio 2019 version 16.5 or later](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&ch=pre&rel=16) for any designer work you want to do. We recommend to update to the [latest version of Visual Studio](https://visualstudio.microsoft.com/vs/).
```
Comment (Youssef1313) [inline docs/core/porting/winforms.md:23]:
```suggestion
- [Visual Studio 2019 version 16.5 or later](https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=community&rel=16) for any designer work you want to do. We recommend to update to the [latest version of Visual Studio](https://visualstudio.microsoft.com/vs/).
```

File: docs/core/porting/winforms.md (hunk 1 above)
Comment (adegeo) [reply docs/core/porting/winforms.md:33]: No I believe this remains, I checked the latest preview and the option is still listed under preview features, so I think it's good to stay for now. However, we can update references from core 3.0 to core 3.1

**Review by adegeo — APPROVED** [review]

**Comment by adegeo** [comment]
Once @gewarren approves, we can merge. Thanks @Youssef1313 !

**Comment by gewarren** [comment]
> Once @gewarren approves, we can merge. Thanks @Youssef1313 !

I think we need to update 3.0 to 3.1 and remove the TODO comment, right?

**Comment by adegeo** [comment]
Thinking about it some more, I think the todo can be removed. This article will be evaluated and updated when .NET 5 releases or the desktop guide is released.

**Comment by Youssef1313** [comment]
> I think we need to update 3.0 to 3.1

@gewarren, Wouldn't 3.x be more accurate?

File: docs/core/porting/winforms.md [hunk 4]
```diff
@@ -31,7 +31,7 @@ In this article, various names are used to identify types of files used for migr
 - A project coded in C#.
 
 > [!NOTE]
-> .NET Core 3.0 projects are only supported in **Visual Studio 2019** or a later version. Starting with **Visual Studio 2019 version 16.5 Preview 1**, the .NET Core Windows Forms designer is also supported.
+> .NET Core 3.0 projects are only supported in **Visual Studio 2019** or a later version. Starting with **Visual Studio 2019 version 16.5**, the .NET Core Windows Forms designer is also supported.
```
Comment (gewarren) [inline docs/core/porting/winforms.md:34]:
```suggestion
> .NET Core Windows Forms projects are supported in Visual Studio 2019 and later versions. The .NET Core Windows Forms designer is supported starting in Visual Studio 2019 version 16.5.
```

I changed the wording to remove 3.0 completely. @adegeo Is this okay?

**Pull request merged**